cmd/cloud/group.go

```diff
@@ -261,6 +265,33 @@ var groupListCmd = &cobra.Command{
 	},
 }
 
+var groupGetStatusCmd = &cobra.Command{
+	Use:   "status",
+	Short: "Get a particular groups status.",
+	RunE: func(command *cobra.Command, args []string) error {
+		command.SilenceUsage = true
+
+		serverAddress, _ := command.Flags().GetString("server")
+		client := model.NewClient(serverAddress)
+
+		groupID, _ := command.Flags().GetString("group")
+		groupStatus, err := client.GetGroupStatus(groupID)
+		if err != nil {
+			return errors.Wrap(err, "failed to query group status")
+		}
+		if groupStatus == nil {
+			return nil
+		}
+
+		err = printJSON(groupStatus)
+		if err != nil {
+			return err
```

Comment:
Could you `Wrap` this `err` with some context?

Reply:
I see those errors from `printJSON` are not wrapped anywhere else in the whole client, so I wonder if it is not better to keep it consistent.
If that is a mistake on the other hand I can add wrap in the whole file.

Reply:
Eh, let's keep it consistent for now.